internal/util/projutil/project_util.go

```diff
@@ -351,3 +352,17 @@ func appendContent(fileContents, target, newContent string) (string, error) {
 	return fileContents[:index] + newContent + fileContents[index:], nil
 
 }
+
+// GetOperatorName returns the name of the operator which is by default the projectName attribute of the PROJECT file
+// However, the Go projects built with the plugin version v2 has not this attribute and then, for this case
+// the operatorName will be the current directory.
+func GetOperatorName(cfg *config.Config) (string, error) {
```

Comment:
This function is only ever used in `generate` so move this to `cmd/operator-sdk/generate/internal/genutil.go`.